js/controllers/list/submissions/SubmissionsListItem.vue

```diff
@@ -212,6 +212,14 @@ export default {
 							notice = this.activeStage.status;
 							break;
 					}
+					if (!this.activeStage.currentUserCanRecommendOnly) {
+						switch (this.activeStage.statusId) {
+							case 13: // REVIEW_ROUND_STATUS_RECOMMENDATIONS_READY
+							case 14: // REVIEW_ROUND_STATUS_RECOMMENDATIONS_COMPLETED
```

Comment:
We just merged a new approach for handling constants in the `.vue` components here: https://github.com/pkp/pkp-lib/issues/2708

Basically, add the constant to the constants array in `SubmissionListHandler::getConfig()`. Then you should be able to reference it at `pkp.const.CONST_NAME`.